feature_engine/encoding/count_frequency.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         encoding_method: str = "count",
         variables: Union[None, int, str, List[Union[str, int]]] = None,
         ignore_format: bool = False,
+        rare_labels: str = "ignore"
```

Comment:
we need to add this parameter in the docstrings. After line 56. Could you add that?

For inspiration, look at the doscrtings of the param "errors" in pandas to datetime: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_datetime.html